### ISSUE TITLE
JER Anvil Recipe removes enchantments

### ISSUE THREAD

**Issue by smmmadden** [NerdyPuzzle/Just-enough-recipes-MCreator#3]
I created a JER Recipe for repairing tools on the Anvil.  What happens is if the tool has enchantments, they get removed during the repair.  The expectation is if a tool has enchantments and being repaired, it should keep all the enchantments AND repair the tool.  Perhaps the Anvil Recipe either needs to include a checkbox for Keep Enchantments or automatically check existing enchantments and maintain post-repair.
Recipe for repair Painite Swords
![image](https://github.com/NerdyPuzzle/Just-enough-recipes-MCreator/assets/10309346/86ef3ee5-fc9b-4fb0-84a1-9d1aabeaf952)
Current Enchantments and needing repair
![image](https://github.com/NerdyPuzzle/Just-enough-recipes-MCreator/assets/10309346/0d52c5b4-bbdc-4dd5-ac84-a4cd0d0cdabd)
Anvil Repair Screen for Painite Sword and right-side showing no enchantments
![image](https://github.com/NerdyPuzzle/Just-enough-recipes-MCreator/assets/10309346/0a760809-f422-4ec0-8442-bad597da1db1)
Sword repaired and now with no enchantments
![image](https://github.com/NerdyPuzzle/Just-enough-recipes-MCreator/assets/10309346/3a856240-7d1c-4060-9155-24739dbf0d72)

I also tried pickaxes as well and they lose the enchantments.  However, my armor retains them so could just be tools not being retained.
Thanks - Steve

**Comment by NerdyPuzzle** [comment]
You should use the repair item fields in the tool element not an anvil recipe. Either way, I will look into adding a checkbox to keep the nbt of the item being used in the recipe.

**Comment by smmmadden** [comment]
Agreed, but it doesn't work.  For some reason MCreator still doesn't support it.  I've tested my swords, pickaxes and bow and all three lose their enchantments.  Each of the tools have the repair items present, but without the Anvil recipe, it doesn't register.  That's why I was very happy to see you had the anvil recipe option.  :-)

**Comment by NerdyPuzzle** [comment]
Seems you figured it out anyway
